roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -107,30 +107,27 @@ def get_var(self, *keys, **kwargs):
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
 
-        convert = kwargs.get("type")
+        convert = kwargs.get("convert", lambda x: x)
         try:
-            if convert is None:
-                pass
-            elif convert in ["str", str]:
-                return str(value)
-            elif convert in ["int", int]:
-                return int(value)
-            elif convert in ["float", float]:
-                return float(value)
-            elif convert in ["bool", bool]:
+            if convert is bool:
                 return str(value).lower() in ["true", "yes", "1"]
```

Comment:
I forgot about `y|n` and `on|off`.

Ansible is sort of a mishmash of ini and yaml, since you have the inventory as well as code, group variable files, etc.

My intuition is less about following a standard and more about not surprising the users. Which is a little bit chicken and egg, since following a standard should lead to consistent user expectations, in theory.

In the end, we should probably not implement our own here, and just re-use whatever the jinja `bool` filter already does. Then it will at least be consistent with what the rest of Ansible does, and "not our problem."